in_text/text_data_release.yml

```diff
@@ -104,58 +134,66 @@ entities:
     attributes:
     - 
       attr-label: time
-      attr-def: Calendar date
-      attr-defs: NA
+      attr-def: Date of this weather estimate.
+      attr-defs: Read et al., 2021
       data-min: NA
       data-max: NA
       data-units: NA  
     - 
       attr-label: ShortWave	
-      attr-def: Daily average incoming shortwave radiation
-      attr-defs: NA
+      attr-def: >- 
+        Incoming radiant energy with wavelengths between ~300 and ~3000 nm, emitted by the 
+        sun and reaching the lake surface after passing through the atmosphere, as a daily average.
+      attr-defs: Read et al., 2021
       data-min: 0
       data-max: NA
       data-units: W m^-2
     - 
       attr-label: LongWave	
-      attr-def: Daily average incoming longwave radiation
-      attr-defs: NA
+      attr-def: >-
+        Incoming thermal radiation at wavelengths > 3000 nm directed from the atmosphere to 
+        the lake surface, as a daily average.
+      attr-defs: Read et al., 2021
       data-min: NA
       data-max: NA
       data-units: W m^-2	
     - 
       attr-label: AirTemp	
-      attr-def: Daily average air temperature
-      attr-defs: NA
+      attr-def: Air temperature above the surface of the lake, as a daily average.
+      attr-defs: Read et al., 2021
       data-min: NA
       data-max: NA
       data-units: degrees C		
     - 
       attr-label: RelHum	
-      attr-def: Daily average relative humidity
-      attr-defs: NA
+      attr-def: Relative humidity of the air above the surface of the lake, as a daily average.
+      attr-defs: Read et al., 2021
       data-min: 0
       data-max: 100
       data-units: >-
         %
     - 
       attr-label: WindSpeed	
-      attr-def: Daily average windspeed
-      attr-defs: NA
+      attr-def: >- 
+        Speed of the wind above the lake, calculated to a daily number using the methods 
+        described in Winslow et al., 2017.
+      attr-defs: Read et al., 2021
       data-min: 0
       data-max: NA
       data-units: m s^-1 	
     - 
       attr-label: Rain	
-      attr-def: Daily cumulative precipitation as rain
-      attr-defs: NA
+      attr-def: >-
+        Daily rate of rain reaching the lake surface in a day as described in 
+        Winslow et al. 2017; rate as determined by Hipsey et al. 2019.
+      attr-defs: Read et al., 2021
       data-min: 0
       data-max: NA
       data-units: m day^-1 	
   -
     data-name: temperature_obs.zip
```

Comment:
Question: should this file have a `site_id` column for easy matching? I know it is in the filename, but this would make it potentially more usable.

Reply:
I would like to keep this "as is". While I agree that adding the `site_id` to the field data would be more useful in a general sense, having this column added is not more useful in a "run a GLM model to generate a result" sense. The reason for this is that several functions in `glmtools` are written in such a way that the field data is expected to have 3 columns (datetime, depth, variable). Here's an example of the error that pops up when you add in an additional "name" column:

```
# load useful libraries
library(glmtools)
library(tidyverse)

# load the example from `compare_to_field` in `glmtools` package
nc_file <- system.file("extdata", "output/output.nc", package = "glmtools")
field_file <- system.file("extdata", "LakeMendota_field_data_hours.csv", package = "glmtools")

# this works!
thermo_values <- compare_to_field(nc_file, field_file,
                                  metric = 'thermo.depth', as_value = TRUE)


# reformat the example data - adding a "name" column and saving to a tmp directory
tmp_dir <- tempdir()
chk <- readr::read_csv(field_file)
chk_alt <- chk %>% 
  mutate(name = 'fake_name') %>% 
  relocate(name)

write_csv(chk_alt, file = file.path(tmp_dir, "field_data_w_name.csv"))

# this errors!
thermo_values <- compare_to_field(nc_file, file.path(tmp_dir, "field_data_w_name.csv"),
                                  metric = 'thermo.depth', as_value = TRUE)
```

The error:
```
Error in resample_to_field(nc_file, field_file, ...) : 
  no field data overlap with simulation period
In addition: Warning message:
In resample_sim(glm_temp, t_out, ...) :
  no matches found using method = 'match' at hours precision. Try method = 'interp'
```

We could work around this, but I would have to revisit the modeling repo.